nix/default.nix

```diff
@@ -13,6 +14,7 @@
   libxcb,
   libxkbcommon,
   mesa,
+  makeWrapper,
```

Comment:
I'd put this between `pkg-config` and `meson`, since it's a `nativeBuildInput`. I tend to put those higher in the set.

Reply:
alright, wasn't sure how it was organized so I just put them in where it seemed correct